flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -64,4 +68,15 @@ public void testConfigurationOfParallelism() {
 		assertEquals(parallelism, config.getParallelism());
 	}
 
+	@Test(expected = UnsupportedOperationException.class)
```

Comment:
I think this test would also pass of line 75 already throws the exception.

I personally discourage the `@Test(expected = )` pattern completely, as you have no check which statement actually throws the exception.